ocaml/tests/test_certificates.ml

```diff
@@ -181,12 +185,15 @@ let invalid_leaf_cert_tests =
   List.map test_invalid_leaf_cert invalid_leaf_certificates
 
 let test_valid_cert_chain chain time pkey () =
-  validate_certificate Chain chain time pkey
+  let _leaf = validate_certificate Chain chain time pkey in ()
```

Comment:
sometimes you use an explicit type below, I think that is better.
Otherwise you could've just used ignore, but using explicit type is preferable to make sure you actually call the function, and not just curry it.